server/controllers/gemini.ts

```diff
@@ -128,4 +128,34 @@ export async function continueStory(ctx: Koa.Context) {
   }
 }
 
+export async function addMoreStory(ctx: Koa.Context) {
+  const profileId = parseInt(ctx.params.profileId);
+  const storyId = parseInt(ctx.params.storyId);
+  const { title, storyString, prompt, readingTime, readingLevel, themes } = ctx.request.body as CreateStoryRequestBody
```

Comment:
I would get all of these from the DB instead since you are already accessing it on line 150